### DISCUSSION TITLE
XML Advanced Electronic Signatures (XAdES)

### DISCUSSION THREAD

**Discussion by fmoric** [microsoft/ALAppExtensions#21877] [Ideas]
Could it be possible to implement XAdES (https://www.w3.org/TR/XAdES/) in cryptography management? In Europe it is widely used for signing eDocuments, almost every country in EU is using some form of it.

**Comment by darjoo** [comment]
Hi @fmoric, it would be possible and this is something we'd take in as a partner contribution. :)

**Reply by fmoric** [reply]
Sure, I will warm up the idea.

I got one query, I read on forums that MS France had developed XAdES Starter Kit. But there is no official, repository to be found.
I found source code in one project https://github.com/nonorganic/dssnet/tree/master/Microsoft.Xades. Can this be verified? Or can we get a hold of the original?
